docs/dev-guide/integration/rest-api/tasks.rst

```diff
@@ -119,6 +119,30 @@ All currently running and waiting tasks may be listed. This returns an array of
 
 | :return:`array of` :ref:`task_report`
 
+
+Deleting Completed Tasks
+------------------------
+
+All completed tasks with states *finished, error, timed out, skipped* may be deleted.
+This call returns response code 204 if successful or code 403 if the request is forbidden.
+
+| :method:`delete`
+| :path:`/v2/tasks/`
+| :permission:`delete`
+
+* :param:`?state,str,only delete tasks with state as state parameters`
```

Comment:
Can the valid values for this param be listed here? I see them above but it might be clearer as part of the param listing